operator/v1/types_image_content_source_policy.go

```diff
@@ -0,0 +1,99 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ImageContentSourcePolicy holds cluster-wide information about how to handle registry mirror rules.
+// When multiple policies are defined, the outcome of the behavior is defined on each field.
+//
+// +kubebuilder:storageversion
+// Compatibility level 1: Stable within a major release for a minimum of 12 months or 3 minor releases (whichever is longer).
+// +openshift:compatibility-gen:level=1
+type ImageContentSourcePolicy struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// spec holds user settable values for configuration
+	// +kubebuilder:validation:Required
+	// +required
+	Spec ImageContentSourcePolicySpec `json:"spec"`
+}
+
+// ImageContentSourcePolicySpec is the specification of the ImageContentSourcePolicy CRD.
+type ImageContentSourcePolicySpec struct {
```

Comment:
as this was pre-existing as v1alpha1, what is the plan about conversion and deprecation?

Reply:
A Jira card was created to record the deprecation [OCPNODE-717](https://issues.redhat.com/browse/OCPNODE-717). The repositories currently use v1alpha1 ImageContentSourcePolicy will be migrated to v1.